src/audio-worklet/basic/volume-meter/main.js

```diff
@@ -3,29 +3,47 @@
 // found in the LICENSE file.
 
 const audioContext = new AudioContext();
+let mediaStream;
+let volumeMeterNode;
 
 const startAudio = async (context, meterElement) => {
   await context.audioWorklet.addModule('volume-meter-processor.js');
-  const mediaStream = await navigator.mediaDevices.getUserMedia({audio: true});
+  mediaStream = await navigator.mediaDevices.getUserMedia({ audio: true });
   const micNode = context.createMediaStreamSource(mediaStream);
-  const volumeMeterNode = new AudioWorkletNode(context, 'volume-meter');   
-  volumeMeterNode.port.onmessage = ({data}) => {
+  volumeMeterNode = new AudioWorkletNode(context, 'volume-meter');
+  volumeMeterNode.port.onmessage = ({ data }) => {
     meterElement.value = data * 500;
   };
   micNode.connect(volumeMeterNode).connect(context.destination);
 };
 
-// A simplem onLoad handler. It also handles user gesture to unlock the audio
+const stopAudio = () => {
+  if (mediaStream) {
+    mediaStream.getTracks().forEach(track => track.stop());
+    mediaStream = null;
+  }
+  if (volumeMeterNode) {
+    volumeMeterNode.disconnect();
+    volumeMeterNode = null;
+  }
+};
+
+// A simple onLoad handler. It also handles user gesture to unlock the audio
 // playback.
 window.addEventListener('load', async () => {
   const buttonEl = document.getElementById('button-start');
   const meterEl = document.getElementById('volume-meter');
   buttonEl.disabled = false;
   meterEl.disabled = false;
+
   buttonEl.addEventListener('click', async () => {
-    await startAudio(audioContext, meterEl);
-    audioContext.resume();
-    buttonEl.disabled = true;
-    buttonEl.textContent = 'Playing...';
+    if (!mediaStream) { // If audio is not playing, start audio
+      await startAudio(audioContext, meterEl);
```

Comment:
Same here.

Reply:
I made the requested changes here:

https://github.com/GoogleChromeLabs/web-audio-samples/pull/362